pkg/image/validation/registry_test.go

```diff
@@ -0,0 +1,238 @@
+package validation
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+)
+
+func TestValidateEmbeddedArtefactRegistry(t *testing.T) {
```

Comment:
Same comment as before, shows up here and on line 65, but only for `2/17` instances.